script.js

```diff
@@ -88,15 +88,12 @@ function onMidi(player, data, time) {
     // NOTE OFF
     const note = hold(data[1], false);
     controls[note].classList.remove("pressed");
-    if (!Object.values(holding).includes(true))
-      player.mute();
```

Comment:
My thinking here is that the audio plays for as long as you're holding the note—kinda simulating a "release". If you delete this, then the video/audio will keep playing until the end right?

Reply:
Oh I gotchu, my bad! Un-deleted this in the latest commit, but changed `mute()` to `pause()` in keeping w/ the other changes 👍